man/networkctl.xml

```diff
@@ -96,8 +96,10 @@
               <varlistentry>
                 <term>degraded-carrier</term>
                 <listitem>
-                  <para>for bond or bridge master, one of the bonding or bridge slave network interfaces is
-                  in off, no-carrier, or dormant state</para>
+                  <para>for bond or bridge master, all of the bonding or bridge slave network interfaces is
+                  in off, no-carrier, or dormant state. As long as at least one of the slave network
+                  interfaces have carrier and the bond or bridge master interace have a routable address
+                  the state remains routable.</para>
```

Comment:
Then, I'd like to make the explanation inverse, i.e. "one of the bonding or bridge slave network interfaces is in off, no-carrier, or dormant state, and the master interface has no address."

Also, please update the explanation for degraded and routable.

Reply:
done that now, altough some one more well spoken should probably write a better explanation :)